Dockerfile

```diff
@@ -4,29 +4,20 @@
 ##
 #
 
-# Let's start with a fresh debian jessie
-FROM debian:jessie
+FROM debian:latest
```

Comment:
Using `latest` can be a problem when a new debian  with some breaking change is release

Reply:
A smaller image can probably be achieved when using Alpine as a base. Since the Dockerfile is just copying build binaries there should be no real difference in the resulting functionality.

Reply:
I actually needed to compile the dependencies statically into the binary, since Alpine [does not support race tests](https://github.com/golang/go/issues/14481) as build env.